dev/ci/periodics/run-evals

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+set -x
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd ${REPO_ROOT}
+
+
+if [[ -z "${OUTPUT_DIR:-}" ]]; then
+    OUTPUT_DIR="${REPO_ROOT}/.build/k8sbench"
+    mkdir -p "${OUTPUT_DIR}"
+fi
+echo "Writing results to ${OUTPUT_DIR}"
+
+BINDIR="${REPO_ROOT}/.build/bin"
+mkdir -p "${BINDIR}"
+
+cd "${REPO_ROOT}"
+go build -o "${BINDIR}/kubectl-ai" .
+
+cd "${REPO_ROOT}/k8s-bench"
+go build -o "${BINDIR}/k8s-bench" .
+
+# evaluate model performance for scale related tasks
```

Comment:
nit: comment is not relevant because the following command runs all tasks.

Reply:
Doh, thanks - will fix next time I touch this file :-)